graylog2-web-interface/src/views/components/sidebar/ViewDescription.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import type { ViewMetaData } from 'views/stores/ViewMetadataStore';
+
+import { SearchResultOverview } from 'views/components/sidebar';
+
+const Section = styled.div`
+  margin-bottom: 8px;
+`;
+
+const defaultNewViewSummary = 'No summary.';
+
+type Props = {
+  results: Object,
+  viewMetadata: ViewMetaData,
+};
+
+const ViewDescription = ({ results, viewMetadata }: Props) => {
+  const formatViewDescription = (view: ViewMetaData) => {
+    const { description } = view;
+    if (description) {
+      return <span>{description}</span>;
+    }
+    return <i>No view description.</i>;
```

Comment:
A search does not have a `description` and will not get any.. so you could make a `<IfDashboard />` arround it somehow and fix: #6725